changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 3.4.11 (2019-10-04)
+  * Fix - WhooCommerce built-in checkout use Billmates new logos
```

Comment:
Should be:

Fix - WooCommerce built-in Checkout are using Billmate's new logos